presto-parquet/src/main/java/io/prestosql/parquet/writer/ParquetWriters.java

```diff
@@ -165,9 +166,12 @@ private static PrimitiveValueWriter getValueWriter(ValuesWriter valuesWriter, io
         if (DATE.equals(type)) {
             return new DateValueWriter(valuesWriter, parquetType);
         }
-        if (BIGINT.equals(type) || TIMESTAMP.equals(type)) {
+        if (BIGINT.equals(type)) {
```

Comment:
Move this above, after the other integer types

Reply:
Moved.